backend/src/main/java/de/neuefische/backend/service/ToDoService.java

```diff
@@ -0,0 +1,54 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.model.ToDo;
+import de.neuefische.backend.model.ToDoInput;
+import de.neuefische.backend.totodao.ToDoDao;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class ToDoService {
+
+    private ToDoDao toDoDao;
+
+    @Autowired
+    public ToDoService(ToDoDao toDoDao) {
+        this.toDoDao = toDoDao;
+    }
+
+    public Optional<List> getAllToDos() {
```

Comment:
```suggestion
    public List<ToDo> getAllToDos() {
```
when your list ist empty just return an empty list